CHANGELOG.md

```diff
@@ -1,6 +1,8 @@
 ## 0.6.0 
 ### Bug Fixes
 
+### Plan to deprecate
```

Comment:
nit: let's use format similar to one in Python client 
https://github.com/ClickHouse/clickhouse-connect/blob/main/CHANGELOG.md?plain=1#L3-L5

Reply:
in python, it's a global message do we want to associate it with a version of